packages/react-native-worklets/src/shareables.ts

```diff
@@ -33,9 +33,13 @@ function isHostObject(value: NonNullable<object>) {
   return MAGIC_KEY in value;
 }
 
-export function isShareableRef(value: unknown): value is ShareableRef {
+export function isSerializableRef(value: unknown): value is SerializableRef {
+  'worklet';
   return (
-    typeof value === 'object' && value !== null && '__shareableRef' in value
+    typeof value === 'object' &&
+    value !== null &&
+    '__serializableRef' in value &&
+    value.__serializableRef === true
```

Comment:
Let's not change the behavior in renaming PR.

```suggestion
    '__serializableRef' in value
```